apis/resourcecontrollerv2/v1alpha1/resourcekey_types.go

```diff
@@ -17,30 +17,26 @@ limitations under the License.
 package v1alpha1
 
 import (
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// In spec mandatory fields should be by value, and optional fields pointers
-// In status, all fields should be by value, except timestamps - metav1.Time, and runtime.RawExtension which requires special treatment
-// https://github.com/crossplane/crossplane/blob/master/design/one-pager-managed-resource-api-design.md#pointer-types-and-markers
-
 // ResourceKeyParameters are the configurable fields of a ResourceKey.
 type ResourceKeyParameters struct {
-	// The human-readable name of the key.
-	Name string `json:"name,omitempty"`
+	// The name of the key.
+	Name string `json:"name"`
```

Comment:
I think we do _not_ want `omitempty` on this property, since it is required by the API -- right?

Reply:
right, my mistake